helm-charts/konk-operator/templates/crds.yaml

```diff
@@ -0,0 +1,6 @@
+{{- if .Values.crds.create }}
+{{- range $path, $bytes := .Files.Glob "crds/*.yaml" }}
```

Comment:
`crds/*.yaml` is symlinked to `config/crd/bases`, which is empty now, so I don't think this will work.

Reply:
looks like we can't symlink, package is also failing